truffle/contracts/Master.sol

```diff
@@ -3,37 +3,23 @@ pragma solidity ^0.4.10;
 import "./Energy.sol";
 
 contract Master {
-    struct ContractEntity {
-        address contract_addr;
-        address seller;
-        bool deregistered;
-    }
 
-    ContractEntity[] public contracts;
-    // lookup index of contract in the list
-    mapping (address => uint) lookupIdxByContract;
+    address[] public contracts;
+    event Update(uint index);
```

Comment:
Is there a reason why this is `Update` and not `update like in the other contract?

Reply:
i need to change other ones, CapCamelCase is recommended for events

Reply:
will approve after that change

Reply:
/noUpdate/empty/s ? you want to indicate contract can be discarded, noUpdate doesn't say that

Reply:
good point